### PR TITLE
Python 3 compatibility for workunits

### PR DESCRIPTION
**Pull request by onyb**

### CODE REVIEW

File: qa/workunits/fs/misc/direct_io.py [hunk 1]
```diff
@@ -34,14 +36,15 @@ def main():
...
     os.lseek(fd, 0, os.SEEK_SET)
-    out = os.read(fd, 1);
+    out = os.read(fd, 1)
     if out != '2':
         raise RuntimeError("data were not directly read from object store")
 
-    os.close(fd);
-    print 'ok'
+    os.close(fd)
+    print('ok')
```
Comment (tchaikov) [inline qa/workunits/fs/misc/direct_io.py]: do we need 
```py
from __future__ import print_function
```
for this?

Comment (onyb) [inline qa/workunits/fs/misc/direct_io.py]: @tchaikov Since the `print` statement is a very simple one, we don't need that line.

Comment (tchaikov) [inline qa/workunits/fs/misc/direct_io.py]: oic, `print()` is introduced in py2.6, and we've bumped the supported python version to >= 2.6

**Comment by tchaikov** [comment]
and needs to run through ceph-disk and cephfs qa suites

**Comment by tchaikov** [comment]
ceph-disk test passed at http://pulpito.ceph.com/kchai-2016-09-19_07:05:39-ceph-disk-wip-kefu-testing---basic-vps/

**Pull request merged**